projects/igniteui-angular/src/lib/simple-combo/README.md

```diff
@@ -0,0 +1,317 @@
+    </ng-template>
+</igx-simple-combo>
+```
+
+#### Defining toggle icon template:
+
+```html
+<igx-simple-combo>
+    <ng-template igxComboClearIcon>
+        <igx-icon>clear</igx-icon>
+    </ng-template>
+</igx-simple-combo>
+```
+
+## Keyboard Navigation
+
+When the combo is closed and focused:
+- `ArrowDown` or `Alt` + `ArrowDown` will open the dropdown and will move focus to the selected item, if no selected item is present, the first item in the list wil lbe focused.
+
+When the combo is opened:
+- `ArrowUp` will close the dropdown and will move focus to the input while also selecting all of the text in the input if the currently focused item is the first item in the list. Otherwise `ArrowUp` will move to the previous list item.
+- `ArrowDown` will move focus from the search input to the first list item. If list is empty and custom values are enabled will move it to the Add new item button.
+- `Alt` + `ArrowUp` will close the dropdown.
+
+When the combo is opened and a list item is focused:
+- `End` will move to last list item.
+- `Home` will move to first list item.
+- `Space` will select/deselect active list item without closing the dropdown.
+- `Enter` will confirm the currently focused item as selected and will close the dropdown.
+- `Esc` will close the dropdown.
+
+When the combo is opened, allow custom values are enabled and add item button is focused:
+- `Enter` will add new item with `valueKey` and `displayKey` equal to the text in the input and will select the new item.
+- `ArrowUp` will move the focus back to the last list item or if the list is empty will move it to the input.
+
+<div class="divider--half"></div>
+
+## Display Density
+**igx-simple-combo** supports setting of different display densities.
+Display density is received through Angular's DI engine or can be set through the `[displayDensity]` input. The possible display densities are `compact`, `cosy` and `comfortable` (default).
+Setting `[displayDensity]` affects the control's items' and inputs' css properties, most notably heights, padding, font-size.
+
+### Properties
+| Name                     | Description                                       | Type                        |
+|--------------------------|---------------------------------------------------|-----------------------------|
+| `id`                    | combo id                                          | `string`                      |
+| `data`                  | combo data source                                 | `any`                         |
+| `value`                 | combo value                                       | `string`                      |
+| `allowCustomValue`      | enables/disables combo custom value                | `boolean`                     |
+| `valueKey`              | combo value data source property                  | `string`                      |
+| `displayKey`            | combo display data source property                | `string`                      |
```

Comment:
use same description as in the component
_Determines which column in the data source is used to determine the display value._